docs/cli/build/in-obs.md

```diff
@@ -0,0 +1,53 @@
+---
+title: in-obs
+date: 2024-06-27
+sidebar_position: 5
+---
+
+# `packit build in-obs`
+
+Submit a build of the present content in the upstream or local repository to  [OBS](https://build.opensuse.org/).
+
+## Requirements
+
+* SPEC file for the project placed in the repository.
+* Packit config file placed in the repository.
+
+
+
+## Tutorial
+
+1. [Place a config file for packit in the root of your upstream repository.](/docs/configuration/)
+
+2. The command below would create a SRPM from the present content of a repo and perform `osc commit` with it. 
+:::note
+You should specify a project `--project` else a project name in this form `home:<me>:packit` would be used. Don't expect to see that project on your [OBS](https://build.opensuse.org/) account .
+:::
+    ```
+    $ cd my/ustream/project/
+    $ packit build in-obs --project <project_name>
+    ```
+
+## Help
+
+    Usage: packit build in-obs [OPTIONS] [PATH_OR_URL]
+
+      Build selected project in OBS
+
+    Options:
+      --owner TEXT         OBS user, owner of the project. (defaults to the
+                          username from the oscrc)
+      --project TEXT       Project name to build in. It will be created if does
+                          not exist. It defaults to home:$owner:packit:$pkg
+      --targets TEXT       Comma separated list of chroots to build in. (defaults
+                          to 'fedora-rawhide-x86_64')
+      --description TEXT   Description of the project to build in.
+      --upstream-ref TEXT  Git ref of the last upstream commit in the current
+                          branch from which packit should generate patches (this
+                          option implies the repository is source-git).
+      --wait / --no-wait   Wait for the build to finish
+      -p, --package TEXT   Package to build, if more than one available, like in a
+                          monorepo configuration. Use it multiple times to select
+                          multiple packages.Defaults to all the packages listed
+                          inside the config.
+      -h, --help           Show this message and exit.
```

Comment:
I am not so sure about the usefulness of this section. It just duplicates the help output and will eventually become outdated. I'd honestly just mention that you can run `packit build in-obs --help` to obtain this help.

Reply:
All the build commands have a similar format. Check the docs for [in-koji](https://packit.dev/docs/cli/build/in-koji)

Reply:
Fair enough